server/plugin/plugin.go

```diff
@@ -680,7 +680,7 @@ func (p *Plugin) checkOrg(org string) error {
 	config := p.getConfiguration()
 
 	configOrg := strings.TrimSpace(config.GitHubOrg)
-	if configOrg != "" && configOrg != org {
+	if configOrg != "" && configOrg != org && strings.ToLower(configOrg) != org {
```

Comment:
Should we also cast `org` to lower?

Reply:
@mickmister We have already did it in previous function